rollup/cmd/mock_app.go

```diff
@@ -30,11 +28,11 @@ type MockApp struct {
 	args []string
 }
 
-// NewRollupApp TODO function will be replaced by NewRollupApp2
-func NewRollupApp(base *docker.App, file string) *MockApp {
-	rollupFile := fmt.Sprintf("/tmp/%d_rollup-config.json", base.Timestamp)
+// NewRollupApp return a new rollupApp manager, name mush be one them.
```

Comment:
What this comment mean?

Reply:
This is the comment. Do you need to modify it?

![image](https://github.com/scroll-tech/scroll/assets/38887641/ffdd4d74-9b6d-450a-bc36-d8095bd3a263)

Reply:
done